app/transcribe/gpt_responder.py

```diff
@@ -138,7 +138,7 @@ def generate_response_from_transcript_no_check(self) -> str:
     def create_client(self, api_key: str):
```

Comment:
Add the parameter base_url here as well.
So change the line 
`create_client(self, api_key: str)`
to 
`create_client(self, api_key: str, base_url: str = None)`